app/backend/src/couchers/servicers/donations.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+import grpc
+
+from couchers import config, errors, urls
+from couchers.db import session_scope
+from couchers.models import User
+from pb import donations_pb2, donations_pb2_grpc
+
+enabled = config.config["ENABLE_DONATIONS"]
+
+if enabled:
+    import stripe
+
+    stripe.api_key = config.config["STRIPE_API_KEY"]
+    RECURRING_DONATION_ID = config.config["STRIPE_RECURRING_PRODUCT_ID"]
+
+
+class Donations(donations_pb2_grpc.DonationsServicer):
+    def InitiateDonation(self, request, context):
+        if not enabled:
+            context.abort(grpc.StatusCode.UNAVAILABLE, errors.DONATIONS_DISABLED)
+
+        with session_scope() as session:
+            user = session.query(User).filter(User.id == context.user_id).one()
+
+            if request.amount < 2:
```

Comment:
If the amount is in cents, this checks for requests < 2 cents, while an error message above seems to say that you want to check for < 2 dollars. I think this should be < 200, or you save the output of ` request.amount * 100` in a variable and use that.

Reply:
Scratch that, the code makes sense, my comment is wrong.